app/assets/stylesheets/content.scss

```diff
@@ -33,9 +54,9 @@
 }
 
 #upvoted {
-  color: #BDD860;
+  color: #bdd860;
 }
 
 #downvoted {
-  color: #D85B36;
+  color: #d85b36;
```

Comment:
Color literals like `#d85b36` should only be used in variable declarations; they should be referred to via variable everywhere else.